composites/Plugin/SnippetPreview/components/SnippetPreview.js

```diff
@@ -632,6 +645,7 @@ export default class SnippetPreview extends PureComponent {
 					{ ...outerContainerProps }
 				>
 					<MobileDescriptionOverflowContainer
+						isDescriptionGenerated={ this.state.isDescriptionGenerated }
```

Comment:
This is only called when mode is mobile, will it also be called when it is in desktop mode?

Reply:
Yes, it is located in the `outerContainerProps` object. I don't know why the mobile container is wrapped twice, but I think both the `MobileDescriptionOverflowContainer` and the `MobileDescriptionWithCaret` need to have the `isDescriptionGenerated` prop to set the right colors.